assets/js/modules/analytics/components/common/ExistingTagNotice.js

```diff
@@ -220,25 +63,5 @@ export default function ExistingTagNotice() {
 		);
 	}
 
-	if ( ua.hasExistingTag ) {
-		if ( ga4.hasExistingTag ) {
-			notice = getNoticeForExistingUAAndGA4Tags();
-		} else {
-			notice = getNoticeForExistingUATag();
-		}
-	} else if ( hasGTMAnalyticsProperty ) {
-		if ( ga4.hasExistingTag ) {
-			notice = getNoticeForExistingGTMPropertyAndGA4Tag();
-		} else {
-			notice = getNoticeForExistingGTMProperty();
-		}
-	} else if ( ga4.hasExistingTag ) {
-		notice = getNoticeForExistingGA4Tag();
-	}
-
-	if ( ! notice ) {
-		return null;
-	}
-
-	return <p>{ notice }</p>;
+	return <p>{ getNoticeForExistingGTMProperty() }</p>;
```

Comment:
Thanks, @kuasha420. Looks like we don't need the `getNoticeForExistingGTMProperty` function anymore and can simplify the code of this component a bit. Let's remove the function and simply return messages as we usually do in other components:

```js
if ( gtmAnalyticsPropertyID === propertyID ) {
    return <p>{ sprintf( ... ) }</p>;
}

return <p>{ sprintf( ... ) }</p>;
```

This is the last change for this PR. Everything else looks good.

Reply:
Updated. we should be good now.